plugins/json-multisheet-extractor/README.md

```diff
@@ -0,0 +1,78 @@
+<!-- START_INFOCARD -->
+
+The `@flatfile/json-multisheet-extractor` plugin parses a JSON file and extracts second-level nested objects into first level defined Sheets in Flatfile.
+
```

Comment:
Adding an example of the nesting structure here as per https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1825995964